wfe/web-front-end.go

```diff
@@ -380,24 +345,21 @@ func (wfe *WebFrontEndImpl) verifyPOST(logEvent *requestEvent, request *http.Req
 	// *anyway*, so it could always lie about what key was used by faking
 	// the signature itself.
 	if len(parsedJws.Signatures) > 1 {
-		err = core.SignatureValidationError("Too many signatures in POST body")
 		wfe.stats.Inc("WFE.Errors.TooManyJWSSignaturesInPOST", 1, 1.0)
 		logEvent.AddError("too many signatures in POST body: %d", len(parsedJws.Signatures))
-		return nil, nil, reg, err
+		return nil, nil, reg, probs.Malformed("Too many signatures in POST body")
```

Comment:
I think test coverage of errors in the WFE isn't a good proxy for how much we care about the exact error message. More a matter of how strict we were being about tests at the time + not having a good pattern for returning named error cases and so testing for string equality instead of variable equality.